src/plugins/folderView/FolderListView.js

```diff
@@ -35,7 +35,7 @@ define([
             name: 'List View',
             cssClass: 'icon-list-view',
             canView: function (domainObject) {
-                return domainObject.type === 'folder';
+                return domainObject.type === 'folder' || domainObject.type === 'noneditable.folder';
```

Comment:
can we change this to be an array of types, declared at the top of the file? We may need to add new types that can use this view in the future